contrib/dockerd-rootless-setuptool.sh

```diff
@@ -212,22 +212,6 @@ init() {
 		fi
 	fi
 
-	# instruction: RHEL/CentOS 7 requires setting max_user_namespaces
-	if [ -f /proc/sys/user/max_user_namespaces ]; then
```

Comment:
@AkihiroSuda - cleaning up bits / leftovers for CentOS 7 / RHEL 7 - this is no longer needed for RHEL 8, correct? Or still needed?

Reply:
Correct, but it might be still needed on some customized hosts.

So I suggest keeping this check.